### PR TITLE
feat(account): adding account deep link (SQSERVICES-1140)

### PR DESCRIPTION
**Pull request by tlebon**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/SQSERVICES-1140" title="SQSERVICES-1140" target="_blank">StorySQSERVICES-1140</a>  [Web] Display user link in self user profile
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?
https://www.figma.com/file/gdfpbVFEhHQNKOTOqf2lMe/Misc-Design-requests?node-id=14%3A627
### Issues

_Briefly describe the issue you have solved or implemented with this pull request. If the PR contains multiple issues, use a bullet list._

### Causes (Optional)

_Briefly describe the causes behind the issues. This could be helpful to understand the adopted solutions behind some nasty bugs or complex issues._

### Solutions

_Briefly describe the solutions you have implemented for the issues explained above._

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

### Testing

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: src/script/page/preferences/AccountPreferences.tsx [hunk 1]
```diff
@@ -185,6 +186,12 @@ const AccountPreferences: React.FC<AccountPreferencesProps> = ({
                 readOnly
               />
             ))}
+            <AccountLink
+              label={t('preferencesAccountLink')}
+              value={`${Config.getConfig().URL.ACCOUNT_BASE}/user-profile/?id=${selfUser.id}`}
+              readOnly
+              data-uie-name="profile-link"
```
Comment (Yserz) [inline src/script/page/preferences/AccountPreferences.tsx]:
```suggestion
              data-uie-name="element-profile-link"
```
@see https://github.com/wireapp/wire-webapp/wiki/data-attributes

Comment (tlebon) [reply src/script/page/preferences/AccountPreferences.tsx]: updated

File: test/unit_tests/pages/preferences/AccountLinkSpec.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
```
Comment (Yserz) [inline test/unit_tests/pages/preferences/AccountLinkSpec.tsx]:
```suggestion
 * Copyright (C) 2022 Wire Swiss GmbH
```

**Review by Yserz — APPROVED** [review]

File: test/unit_tests/pages/preferences/AccountLinkSpec.tsx (hunk 2 above)
Comment (tlebon) [reply test/unit_tests/pages/preferences/AccountLinkSpec.tsx]: updated

File: src/script/page/preferences/accountPreferences/AccountLink.tsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
+/*
+ * Wire
+ * Copyright (C) 2021 Wire Swiss GmbH
```
Comment (Yserz) [inline src/script/page/preferences/accountPreferences/AccountLink.tsx]:
```suggestion
 * Copyright (C) 2022 Wire Swiss GmbH
```
;)

File: src/script/page/preferences/accountPreferences/AccountLink.tsx [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        data-uie-name="do-copy-profile-link"
+        role="button"
+        onClick={() => copyText(value)}
+        css={{
+          alignItems: 'center',
+          cursor: 'pointer',
+          display: 'flex',
+          flexDirection: 'row',
+          paddingLeft: '8px',
+          paddingTop: '8px',
```
Comment (Yserz) [inline src/script/page/preferences/accountPreferences/AccountLink.tsx]:
```suggestion
          paddingLeft: 8,
          paddingTop: 8,
```

File: src/script/page/preferences/accountPreferences/AccountLink.tsx [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+        }}
+      >
+        <Icon.Copy css={{fill: 'var(--background)', marginRight: '8px'}} data-uie-name="profile-link-icon" />
```
Comment (Yserz) [inline src/script/page/preferences/accountPreferences/AccountLink.tsx]:
```suggestion
        <Icon.Copy css={{fill: 'var(--background)', marginRight: 8}} data-uie-name="profile-link-icon" />
```

File: src/script/page/preferences/accountPreferences/AccountLink.tsx (hunk 3 above)
Comment (tlebon) [reply src/script/page/preferences/accountPreferences/AccountLink.tsx]: missed this one 😉

File: src/script/page/preferences/accountPreferences/AccountLink.tsx (hunk 4 above)
Comment (tlebon) [reply src/script/page/preferences/accountPreferences/AccountLink.tsx]: updated

File: src/script/page/preferences/accountPreferences/AccountLink.tsx (hunk 5 above)
Comment (tlebon) [reply src/script/page/preferences/accountPreferences/AccountLink.tsx]: updated

**Review by Yserz — APPROVED** [review]

**Pull request merged**